src/main/kotlin/com/manimdsl/linearrepresentation/Objects.kt

```diff
@@ -140,16 +140,20 @@ data class InitManimStack(
     val color: String? = null,
     val textColor: String? = null,
     val showLabel: Boolean? = null,
+    val creationStyle: String? = null,
+    val creationTime: Double? = null,
     private var boundary: List<Pair<Double, Double>> = emptyList(),
     private var maxSize: Int = -1
 ) : DataStructureMObject(type, ident, boundary) {
     override var shape: Shape = NullShape
 
     override fun toPython(): List<String> {
+        val creationString = if (creationStyle != null) ", creation_style=\"$creationStyle\"" else ""
+        val runtimeString = if (creationTime != null) ", run_time=$creationTime" else ""
         val python =
             mutableListOf("# Constructing new ${type} \"${text}\"", shape.getConstructor())
         val newIdent = if (showLabel == null || showLabel) "\"$ident\"" else ""
```

Comment:
I'm really sorry I think I didn't spot this when I was reviewing the previous pull request but the `$ident` here should be `$text` or else the label would be wrong. I'm really sorry I should have spotted this earlier today but if you don't mind can you just update it here? Thanks!

Reply:
No worries - I'll do it now!